python/python/geoindex_rs/_rust.pyi

```diff
@@ -33,6 +33,11 @@ class KDTree:
 T = TypeVar("T", bound=np.generic, covariant=True)
 
 class RTree(Generic[T]):
+    @classmethod
```

Comment:
One thing I'm not entirely sure about here is annotations for `__getbuffer__`, since it isn't _actually_ accessible from python - I suppose that's part of the impetus for PEP 688 :shrug:

Reply:
yeah you shouldn't annotate `__getbuffer__`. It's not exposed to python